# Publish a vector of joint temperatures [C] for the arm joints
#
# The order of the temperatures is the same as the order of the joint names as published in the joint states topic
int32[] temps
